rmt-helm/README.md

```diff
@@ -5,44 +5,44 @@ It is tested on K3s but should work on any Kubernetes distribution.
 
 ## Overview
 
-Every component of the stack is deployed in a dedicated container via a
-Helm Chart to ease deployment on top of Kubernetes.
+To deployment on top of Kubernetes easier, each component of the stack is deployed in a dedicated container via a
```

Comment:
'easier' reads weird. This is simply the only possible way to deploy something on kubernetes IMHO.